index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+  </head>
+
+  <body>
+
+    <header style="background-color:#ADD8E6;">
+      <img src="images/adoptadog-logo.svg" alt="white adopt a dog logo"  width="503" height="136">
+      <h1>Seattle's premiere dog adoption agency</h1>
+      <img src="images/cart-icon.svg" alt="shopping cart icon" width="29" height="26">
+      <p>$1234.56</p>
+      <a class="checkout-button" href="checkout.html">checkout</a>
+    </header>
+
+    <nav style="background-color:#C0C0C0">
+      <a href="index.html">Home</a> /
```

Comment:
Since your files are looking REALLY good, maybe you can stretch yourself with some CSS and see if you can generate the / 's via css? It's not a big deal, but a way to make these better.